src/dali/plugin/pair_converter/ccxt.py

```diff
@@ -324,18 +351,17 @@ def find_historical_bar(self, from_asset: str, to_asset: str, timestamp: datetim
                     self.__logger.debug(
                         "Exception from server, most likely too many requests. Making another attempt after 0.1 second delay. Exception - %s", exc
                     )
-                    # logger INFO for retry?
-                    sleep(0.1)
-                    request_count += 3
+                    sleep(10)
```

Comment:
If we need more than 0.1 seconds / 3 retries, then we will need to add throttling with `_REQUEST_DELAYDICT`. This is meant to be a safety valve.

Reply:
I'm not sure exactly what we need here, but it seemed odd to use 0.1 when 10 is used elsewhere. Is there a reason not to keep retries consistent?